parachain/src/wasm_executor/mod.rs

```diff
@@ -37,9 +37,25 @@ const MAX_RUNTIME_MEM: usize = 1024 * 1024 * 1024; // 1 GiB
 const MAX_CODE_MEM: usize = 16 * 1024 * 1024; // 16 MiB
 const MAX_VALIDATION_RESULT_HEADER_MEM: usize = MAX_CODE_MEM + 1024; // 16.001 MiB
 
-/// The execution mode for the `ValidationPool`.
+/// The strategy we employ for isolating execution of wasm parachain validation function (PVF).
+///
+/// For a typical validator an external process is the default and recommended way to run PVF. The
+/// rationale is based on two observations:
+///
+/// (a) PVF is completely under control of parachain developers who may or may not be malicious.
```

Comment:
Isn't it more that we don't trust the `PoV'

Reply:
I think it's both! thanks